### DISCUSSION TITLE
Quarto website does not update

### DISCUSSION THREAD

**Discussion by robpetrosino** [quarto-dev/quarto-cli#9164] [Q&A] [answered]
### Description

Hello everyone, 

I hope someone can help me with my issue: My website does not update, and I can't understand why. 

Repo: [https://github.com/robpetrosino/frequency-attenuation_masked-priming](https://github.com/robpetrosino/frequency-attenuation_masked-priming)
Website: [https://robpetrosino.github.io/frequency-attenuation_masked-priming/](https://robpetrosino.github.io/frequency-attenuation_masked-priming/)

I have tried to push the new commit multiple times, as well as re-initialize the `gh-pages` branch and restart from a brand new template, but the page would always show the previous version of the manuscript (which ends at the section "experiment 1"). The new version is supposed to have more section after that. 

Can someone have a look and let me know?

Thank you in advance!

**Comment by mcanouil** [comment]
First thing obvious are the clearly the invalid labels for your code cells.

- https://github.com/robpetrosino/frequency-attenuation_masked-priming/blob/main/index.qmd#L478

At least use the YAML option instead of mixing two syntaxes (R inline and YAML):

````
```{r}
#| label: "experiment 1 -- RT distributions; errors; minimal trial per condition"
````

Note that the labels should not have special characters, punctuations, spaces or underscores, are they are used for cross-reference.

Also you are defining labels twice using both syntaxes:
![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/7d008470-5a6d-4980-84e3-f93c514bfd09)

In Quarto, a code cell "should" be only ` ```{language} `.
Everything you want to add can be added using YAML syntax, _e.g._, `#| ` in R.

And your document is way too big for us to debug it (more than one thousand lines). Unfortunately, we don't have the luxury of time to follow the usual debugging steps for you:
1. "remove" everything.
2. add back small pieces at each time.
3. render.
4. tweak code when you've found the offending code.

> The new version is supposed to have more section after that.

How do you know? Did you try to render locally and it showed more? If so, are you using the same version in GitHub actions and locally? See [GitHub Actions documentation](https://github.com/quarto-dev/quarto-actions) if you don't know how to change the version.

**Comment by robpetrosino** [comment, marked as answer]
Thanks for you replying. I don't know what that was, but I recreated the whole repo from scratch, and now it is working.
